popup/src/index.ts

```diff
@@ -22,13 +22,18 @@ type Options = {
   onAuthorizationCodeReceived?: AuthorizationCodeCallback;
 };
 
+function utf8_to_b64(str: string): string {
+  return window.btoa(unescape(encodeURIComponent(str)));
+}
+
```

Comment:
TIL `unescape`.

Reply:
Yeah, it's from https://developer.mozilla.org/en-US/docs/Glossary/Base64#solution_1_–_escaping_the_string_before_encoding_it

Reply:
Nice, I'll read it for sure!